src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -125,12 +126,25 @@ foam.CLASS({
       },
       code: function() {
         if ( ! this.stack ) return;
-        this.stack.push({
-          class: 'foam.comics.v2.DAOCreateView',
-          data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
-          config$: this.config$,
-          of: this.data.of
-        }, this.__subContext__);
+
+        if ( this.config.DAOCreateView ){
+          this.stack.push({
+            class: 'foam.comics.v2.DAOCreateView',
+            data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
+            config$: this.config$,
+            of: this.data.of
+          }, this.__subContext__);
+        } else {
+          this.stack.push(
+            eval(this.config.wizardDetailView.class).create({
```

Comment:
Here you are creating a view rather than pushing on a viewspec. This will cause the problem of not being able to go back() to this view.